src/main/java/org/jenkinsci/test/acceptance/po/Build.java

```diff
@@ -131,6 +131,7 @@ public boolean isInProgress() {
         }
 
         JsonNode d = getJson();
+        // TODO use "inProgress" if https://github.com/jenkinsci/jenkins/pull/6829 ever reached the LTS (see also JENKINS-68981)
```

Comment:
Well, no, you should use https://github.com/jenkinsci/jenkins/pull/6829 right away if the field is defined. If it is not, then you may have fall back to the old implementation, with a TODO note to clean that up eventually.